common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -90,6 +95,10 @@ def handle_ajax(self, dispatch, data):
             _, _, traceback_obj = sys.exc_info()  # pylint: disable=redefined-outer-name
             raise ProcessingError, (not_found_error_message, err), traceback_obj
 
+        except TimeExpiredError as err:
+            _, _, traceback_obj = sys.exc_info()  # pylint: disable=redefined-outer-name
```

Comment:
We shouldn't shadow `_`.

Instead, Pylint affords us the following syntax to explicitly mark variables as unneeded:

```python
dummy1, dummy2, traceback_obj = sys.exc_info()
```

We can then remove the pylint pragma.